src/CfnFlip/Libyaml.hs

```diff
@@ -35,20 +40,20 @@ makeSequence inner = do
 
 startsMapOrSequence :: Event -> Bool
 startsMapOrSequence = \case
-  EventMappingStart{} -> True
-  EventSequenceStart{} -> True
+  EventMappingStart {} -> True
+  EventSequenceStart {} -> True
```

Comment:
This is our style? This seems worse

Reply:
> This is our style? 

Fourmolu did this. The **first rule of our style guide** is to follow the auto-formatting for anything it can auto-format.

> **TL;DR**: If it can be automated, it should be automated. If the automated outcome has no objective readability problems, we should accept it.

So, I guess?

> This seems worse

I also don't like it.